Gruntfile.js

```diff
@@ -13,14 +12,20 @@ module.exports = function (grunt) {
         return fs.statSync(modules_dir + file).isDirectory();
     });
 
-     _.each(modules, function (name) {
+    _.each(modules, function (name) {
+        var module_path = name + '.templates';
```

Comment:
Then either the linter hasn't run, or the linting rules need to be updated, because our styleguide is to not allow snake_case